src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -487,6 +493,8 @@ void RaftReplDev::fetch_data_from_remote(std::vector< repl_req_ptr_t > rreqs) {
     builder->FinishSizePrefixed(
         CreateFetchData(*builder, CreateFetchDataRequest(*builder, builder->CreateVector(entries))));
 
+    COUNTER_INCREMENT(m_metrics, fetch_rreq_cnt, 1);
```

Comment:
would suggest add a metrics on 2 more things:
1. Number of entries we are requesting
2. Size of total blocks requested

Reply:
Thought about these two things, didn't do that considering the space. 

These two things will be histogram which could take relatively larger space, we can have them if we are fine.